ddtrace/tracer.py

```diff
@@ -328,6 +330,22 @@ def configure(
         if (collect_metrics is None and runtime_metrics_was_running) or collect_metrics:
             self._start_runtime_worker()
 
+        if asbool(environ.get("DD_TRACE_STARTUP_LOGS") or True):
+            try:
+                info = debug.collect(self)
+            except Exception as e:
+                msg = "Failed to collect start-up logs: %s" % e
+                self._log_compat(logging.WARNING, "- DATADOG TRACER DIAGNOSTIC - %s" % msg)
+            else:
+                if self.log.isEnabledFor(logging.INFO):
+                    msg = "- DATADOG TRACER CONFIGURATION - %s" % debug.to_json(info)
```

Comment:
I'll agree with jd here, not sure what we gain with this vs just `json.dumps(info)`, since `to_json()` doesn't do any additional process, not sure the value the indirection is adding.

unless we had `info = debug.as_json(self)` or something that just gave us json from the beginning, but again... eh

Reply:
I think I was going for encapsulation but since the return type is serializable it shouldn't matter 🤷

Reply:
yeah, I think reading the code the assumption would be that `info` is not directly serializable and lead people to think that `to_json()` does something else to get it to be serializable. Better to have the code be more clear, less hopping around to figure out what is going on, especially if it isn't necessary

Reply:
as long as we have a test case that ensures `info` *can* be serialized as JSON, then we should be good